src/views/tutorial-view/types.ts

```diff
@@ -46,11 +47,18 @@ interface TutorialData
 	nextCollectionInSidebar?: ClientCollectionLite
 }
 interface TutorialViewProps {
-	layoutProps: TutorialSidebarSidecarProps
+	layoutProps: Omit<TutorialSidebarSidecarProps, 'children'>
 	product: Omit<LearnProductData, 'slug'> & {
 		slug: LearnClientProduct['slug'] | 'hcp'
 	} // @TODO clean up the hcp / learn product slug types https://app.asana.com/0/1202097197789424/1202946807363608
 	tutorial: TutorialData
+	outlineItems: OutlineLinkItem[]
+	pageHeading: {
+		slug: string
+		text: string
+	}
+	nextCollection?: ClientCollectionLite | null // if null, it is the last collection in the sidebar order
+	metadata: $TSFixMe
```

Comment:
```suggestion
	metadata: {
	  title: string
	  description: string
	 }
```
Is there a reason you wanted to leave this as a 'fix me'? I see the onboarding page has a different metadata interface, maybe we could just consolidate? Not 100% necessary for this PR.

Reply:
No reason i wanted to leave it, just had to add _something_ to avoid a type error relative to the nothing that was there previously, and didn't start down the path of figuring out what specifically to add 😅 Thanks for the suggestion, that looks great! 🙌